core/gas.c

```diff
@@ -16,7 +16,7 @@ bool isobaric_counterdiffusion(struct gasmix oldgasmix, struct gasmix newgasmix,
 {
 	if (!prefs.show_icd)
 		return false;
-	results->dN2 = get_he(oldgasmix) + get_o2(oldgasmix) - get_he(newgasmix) - get_o2(newgasmix);
+	results->dN2 = get_n2(oldgasmix) - get_n2(newgasmix);
```

Comment:
Doesn't that invert the sign of dN2? I.e. shouldn't that be
```
	results->dN2 = get_n2(newgasmix) - get_n2(oldgasmix);
```
?

Reply:
Oops.